travis.yml

```diff
@@ -0,0 +1,29 @@
+language: node_js
+node_js:
+  - "8.10.0"
+
+cache:
+  directories:
+    - "node_modules"
+
+addons:
+  postgresql: "9.6"
+
+env:
+  global:
+    - CODECLIMATE_REPO_TOKEN=1d83bed34d112e88bca8d0cad3eeaf288b32e7f79453656d43fcaae3de79ce0a
+    - Travis_REPO_TOKEN=PaGRiWt19dxv2dOM-_iKWw
+    - SECRET=esther96
```

Comment:
Hey @davidfrank96 , I can see that the secret key and token are in the travis yml file. We want to try as much as possible to keep these values hidden. I believe TravisCI provides a way to work with environment variables where we can have these values saved there and then reference them here

Reply:
okay noted... will remove that